### ISSUE TITLE
How to set config. margin_list so that the loss function uses Arcface during training.

### ISSUE THREAD

**Issue by LB15642219539** [deepinsight/insightface#2331]
According to my understanding, config. margin_list=(1.0,0.5,0), At this time, the Arcface loss function is used, but in many configuration files config. argin_list=(1.0,0.0,0.4).

**Comment by imCaoQuoc** [comment]
> According to my understanding, config. margin_list=(1.0,0.5,0), At this time, the Arcface loss function is used, but in many configuration files config. margin_list=(1.0,0.0,0.4).

You just need to modify in config folder to config.margin_list = (1.0, 0.5, 0.0)

**Comment by lwhite-sys** [comment]
why
